environments/azure/flyte-core/aks.tf

```diff
@@ -29,3 +40,74 @@ identity {
    }
 }
 
+# Additional pre-requisites for GPU scheduling
+resource "azurerm_kubernetes_cluster_node_pool" "gpu_nodes" {
+count = local.gpu_node_pool_count == 0 ? 0 : 1
+depends_on = [ azurerm_kubernetes_cluster.flyte ]
+name = "gpupool"
+kubernetes_cluster_id = azurerm_kubernetes_cluster.flyte.id
+node_count = local.gpu_node_pool_count
+enable_auto_scaling = true
+min_count           = local.gpu_node_pool_min_count
+max_count           = local.gpu_node_pool_max_count
+vm_size             = local.gpu_machine_type
+os_disk_size_gb    = local.gpu_node_pool_disk_size
+
+#Only used in case you request specific accelerators and/or partitions. Additional configuration may be required.
+# Learn more: https://docs.flyte.org/en/latest/user_guide/productionizing/configuring_access_to_gpus.html
+node_labels = {
```

Comment:
Dont want to overcomplicate things here but could we do something like only setting these labels if `local. accelerator` and `local. partition-size` is not None or something?

Reply:
Agree, I'll try to find a good way to do it